blackjack.py

```diff
@@ -17,12 +17,16 @@ class Card(object):
     def __init__(self, value, suit):
         self.value = value
         self.suit = suit
-        self.points = self.get_card_points()
+        self.points = self.card_points
 
     def __repr__(self):
         return '{} of {}'.format(self.value, self.suit)
```

Comment:
Now that you have `__str__`, you should update `__repr__` to follow the convention you mentioned in an earlier comment (it should be machine-readable).

Reply:
Were you hoping for something more intricate than this? @romainrichard 
```
    def __repr__(self):
        return "'{} of {}'".format(self.value, self.suit)
```
Simply added extra quotes around the return object

Reply:
More like this:
```
    def __repr__(self):
        return "'Card({}, {})'".format(self.value, self.suit)
```
Now a script could run you code and get `"Card('11', 'Hearts')"` and just call `eval()` on it to get the object. No need to parse the output to extract the `value` and `suit`.

The main goal of `__repr__` is to be unambiguous. Putting machine-readable code in it isn't required to achieve that goal but it is sufficient.

Reply:
Oooo, that's pretty. I like it

Reply:
Making the __repr__ return enough to eval a new instance is so slick. I love it. I'm totally stealing that trick.

Reply:
Personally I recommend always creating `__repr__` and if needed create `__str__` (in case the output of `__repr__` isn't super human readable or if you want to format the output in a special way).

For example:
```
class Student(object):
    def __init__(self, name, grade):
        self.name = name
        self.grade = grade

    def __repr__(self):
        return "Student('{}', {})".format(self.name, self.grade)
```
Will give you this:
```
In [2]: a = Student("A", 91)

In [3]: print(a)
Student('A', 91)
```

But maybe you have a list of students and would like to print some sort of table with one row per student, formatted in a way that would make the full result a table.
```
from operator import attrgetter


class Student(object):
    def __init__(self, name, grade):
        self.name = name
        self.grade = grade

    def __repr__(self):
        return "Student('{}', {})".format(self.name, self.grade)

    def __str__(self):
        return "| {:<20} |    {:02d} |".format(self.name, self.grade)


def print_students(students):
    print("--------------------------------")
    print("| Name                 | Grade |")
    print("--------------------------------")
    for student in sorted(students, key=attrgetter("grade"), reverse=True):
        print(student)
    print("--------------------------------")
```

And now you get:
```
In [3]: students = [Student("Student {}".format(i), 10 * i) for i in range(10)]

In [4]: print_students(students)
--------------------------------
| Name                 | Grade |
--------------------------------
| Student 9            |    90 |
| Student 8            |    80 |
| Student 7            |    70 |
| Student 6            |    60 |
| Student 5            |    50 |
| Student 4            |    40 |
| Student 3            |    30 |
| Student 2            |    20 |
| Student 1            |    10 |
| Student 0            |    00 |
--------------------------------
```